utils/io/fileutils/files.go

```diff
@@ -506,12 +484,12 @@ func CopyDir(fromPath, toPath string, includeDirs bool, excludeNames []string) e
 
 // Removing the provided path from the filesystem
 func RemovePath(testPath string) error {
-	if _, err := os.Stat(testPath); err == nil {
-		// Delete the path
-		err = RemoveTempDir(testPath)
-		if err != nil {
-			return errors.New("Cannot remove path: " + testPath + " due to: " + err.Error())
-		}
+	_, err := os.Stat(testPath)
+	// Delete the path
+	err = errors.Join(err, RemoveTempDir(testPath))
```

Comment:
it will always run removetempdir. previously it was only when stat wasn't error

Reply:
I changed